src/pages/components/overview/accessibility-status.mdx

```diff
@@ -0,0 +1,62 @@
+  component has been designed and coded to solve a specific UI problem.
+tabs: ['Components', 'Accessibility Status']
+---
+
+import { Tag } from '@carbon/react';
+import A11yStatus from 'components/A11yStatus';
+
+<PageDescription>
+
+Components are one of the key building blocks of the design system. Their
+systematic reuse helps to create visual and functional consistency across
+products.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Accessibility tag status</AnchorLink>
+<AnchorLink>Accessibility tests</AnchorLink>
+<AnchorLink>All component accessibility status</AnchorLink>
+
+</AnchorLinks>
+
+## Accessibility tag status
+
+For every latest release, Carbon runs tests on all components to meet the
+accessibility requirements. These different statuses report the work that Carbon
+has done in the back end. These tests appear only when the components are
+stable.
+
+| Tag                                     | Meaning                                                                                                                                                                                                             |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <Tag type="green">Tested</Tag>          | When all tests pass and there are no violations when running the automated Equal Access Checker on default state, advanced states, and keyboard navigation.                                                         |
+| <Tag type="blue">Partially tested</Tag> | When some tests pass or are removed, they are either combined into existing tests or substituted with new ones and moved to a different file. Skipped tests stay in the file but are clearly marked with test.skip. |
+| <Tag type="teal">Manually tested</Tag>  | When screen reader tests are conducted manually and validated by a human to ensure quality.                                                                                                                         |
+| <Tag>Not available</Tag>                | Usually, with keyboard navigation tests or screen reader, when this type of test is not available for certain components.                                                                                           |
+| <Tag type="purple">Not yet tested</Tag> | Usually, with experimental or unstable components, tests have not been conducted for either automated or manual testing.                                                                                            |
+
+## Accessibility tests
+
+Accessibility AVT (Accessibility Automated Verification Testing) involves the
+use of automated tools and software to assess the accessibility of a digital
+product against established accessibility standards, such as the Web Content
+Accessibility Guidelines (WCAG).
```

Comment:
Every accessibility test falls into one of four categories. Marked with the tags above, these categories show the status for each of these discrete areas of testing. This additional detail helps clarify exactly what ways a component has been tested for accessibility.